python/src/tests/tasks/test_marker_heatmap.py

```diff
@@ -28,10 +28,10 @@ def load_correct_definition(self):
             "body": {
                 "name": "MarkerHeatmap",
                 "nGenes": 5,
-                "cellSetKey": "louvain",
+                "selectedCellSet": "louvain",
                 "groupByClasses": ["louvain"],
                 "selectedPoints": "All",
-                "hiddenCellSetKeys": []    
+                "hiddenCellSets": []    
```

Comment:
There was a difference between the names being used in the UI and the body being sent to the worker.

I renamed them to fit the ones used in the UI, which I believe are the most accurate ones.